src/hpack/decoder.rs

```diff
@@ -191,14 +194,14 @@ impl Decoder {
             // determined from the first byte.
             match Representation::load(ty)? {
                 Indexed => {
-                    tracing::trace!("    Indexed; rem={:?}", src.remaining());
+                    tracing::trace!(rem = src.remaining(), "    Indexed;");
```

Comment:
we may want to remove the indentation in these event messages --- it's essentially an ad-hoc way of showing the same information that's better captured by the span context. however, i thought it might be worth leaving in because some folks are still consuming these events with `log` rather than `tracing`.

Reply:
The indentation is purely for the sanity of the pure soul consuming the `trace` level logs. If a span provides the same thing, that seems fine (or probably even better).

Reply:
Yeah, I just thought that _some_ users might still be consuming those logs through the `log` crate, so they wouldn't see the span context.

Reply:
Not sure how many people would be broken up by _this_ change. I'd maybe get rid of the extra spaces?